docs/4.4. Behaviour - Access Tokens.md

```diff
@@ -121,15 +121,19 @@ also has "read" access to the specific API, a permission key of "read" MUST also
 object.
 
 Individual AMWA specifications MAY specify additional permissions keys specific to the individual API. These MUST be
-registered in the [AMWA NMOS Parameter Registers][AMWA NMOS Parameter Registers]. It is recommended that new permissions keys are only introduced
-alongside a new version of a given AMWA NMOS API in order to minimise the risk of incompatibility.
+registered in the [AMWA NMOS Parameter Registers][AMWA NMOS Parameter Registers]. It is recommended that new
+permissions keys are only introduced alongside a new version of a given AMWA NMOS API in order to minimise the risk of
+incompatibility.
 
 The `value` corresponding to each permission key is a JSON array containing URL path specifiers that the user is
-permitted to perform the specific request against. Wildcards are permitted in the path specifiers. The means by which
-wildcards may be used for pattern matching SHALL align with Section 2.13 of the [POSIX Standard][POSIX Standard]. It is RECOMMENDED
-that wildcards are used in place of URL parameters (such as specifying resource UUID's) to minimise token size.
-Specific URL parameters MAY be used in path specifiers if defining access to a small number of resources (such as
-a single node or device.)
+permitted to perform the specific request against. The asterisk wildcard ('`*`') is permitted in the path
+specifiers.
+
+A '`*`' wildcard can be used to replace zero or more path characters, and may encompass multiple path segments (e.g.
+`/single/*` and `/single/senders/*/constraints` would both correspond to
+`/single/senders/ea388089-9ffb-4a81-b109-a19da845b3b6/constraints`). It is RECOMMENDED that wildcards are used in
+place of URL parameters (such as specifying resource UUID's) to minimise token size. Specific URL parameters MAY be
```

Comment:
Is the concept of "URL parameters" generally understood (as to opposed to URL _query_ parameters)? I know the RAML uses that term, but I'm not sure we have in docs, so it might be worth spelling it out?

Also, no apostrophe in "UUIDs" please :-) :-)